apps/sms/js/thread_list_ui.js

```diff
@@ -514,13 +520,10 @@ var ThreadListUI = {
       }]
     };
 
-    // Add delete option when list is not empty
-    if (ThreadListUI.noMessages.classList.contains('hide')) {
-      params.items.unshift({
-        l10nId: 'selectThreads-label',
-        method: this.startEdit.bind(this)
-      });
-    }
+    params.items.unshift({
+      l10nId: 'selectThreads-label',
+      method: this.startEdit.bind(this)
+    });
```

Comment:
and because we don't have the if condition anymore, you can directly add it in the previous object params, you don't need to use params.items.unshift.